app/routes/missions.burstcube/route.mdx

```diff
@@ -7,10 +7,6 @@ import logo from './logo.png'
 
 import { Alert } from '@trussworks/react-uswds'
```

Comment:
I think you can remove this import since you removed the component below (unless there is another spot where it's used, but I'm pretty sure it's not the case)